[BUG] Outdated peer dependency @sveltejs/kit@"^1.27.3"

**Describe the bug**
When upgrading to `inngest@3.19.8`, I noticed that `@sveltejs/kit` is pointing to an outdated version, we are already at `2.5.10`. 
Having the latest sveltekit package installed, upgrading thus fails with 

```
npm ERR! Could not resolve dependency:
npm ERR! peer @sveltejs/kit@"^1.27.3" from inngest@3.19.8
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Would it be possible to bump sveltekit to latest? In my initial tries the package seems to build with no issues at least, I didn't have time to test it any further though.

**System info (please complete the following information):**

- OS: Mac
- npm package Version 10.15.0
- Framework Sveltekit
- Platform Vercel

--- a/.changeset/tiny-tips-cover.md
+++ b/.changeset/tiny-tips-cover.md
@@ -1,0 +1,5 @@
+---
+"inngest": patch
+---
+
+Widen `@sveltejs/kit` peer dep range to `>=1.27.3` from `^1.27.3`

--- a/packages/inngest/package.json
+++ b/packages/inngest/package.json
@@ -240,7 +240,7 @@
     "npm": "9.6.4"
   },
   "peerDependencies": {
-    "@sveltejs/kit": "^1.27.3",
+    "@sveltejs/kit": ">=1.27.3",
     "@vercel/node": "^2.15.9",
     "aws-lambda": "^1.0.7",
     "express": "^4.19.2",

